src/api/models.py

```diff
@@ -60,3 +68,21 @@ class CheckLicenseFileUpload(models.Model):
     file = models.FileField(upload_to=user_directory_path)
     result = models.CharField(max_length=128,null=False,blank=False)
     status = models.IntegerField(default=200,blank=False)
+
+class SubmitLicenseModel(models.Model):
+
+    created = models.DateTimeField(auto_now_add=True)
+    owner = models.ForeignKey(User, to_field='id', on_delete=models.CASCADE)
```

Comment:
I have used on_delete=models.CASCADE here and as well as at other places.

Reply:
@victorphoenix3 Give me some time to review this, need to compare both master and python3 branch. In the meantime, you can update the branch with master (some small gitignore and readme changes went in last week). I also don't see travis.yml changes in this branch. The readme should have python3 instructions.